src/FacadeAccountCreation/FacadeAccountCreation.API/Controllers/OrganisationsController.cs

```diff
@@ -100,6 +100,19 @@ public async Task<IActionResult> GetNationIdByOrganisationId(Guid organisationId
         }
     }
 
+    [HttpGet]
+    [Route("nation")]
+    [Consumes("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
```

Comment:
you are not returning BadRequest from any code paths

Reply:
updated this controller to return the right status codes.